_dashboards/get-started/quickstart-dashboards.md

```diff
@@ -0,0 +1,118 @@
+1. Select `x` to close the Add panels window.
+1. View the added panel **[Flights] Delay Buckets**, which is added as the last panel on the dashboard.
+
+    <img src="{{site.url}}{{site.baseurl}}/images/add-panel.png" alt="Add panel to dashboard" width="600"/>
+
+## Try it: Creating a visualization panel
+
+Continuing with the preceding dashboard, you'll create a bar chart comparing the number of cancelled flights and delayed flights to delay type, and then add the panel to the dashboard.
+
+1. Change the default [time range]({{site.url}}{{site.baseurl}}/dashboards/get-started/time-filter/) from **24 hours** to **Last 7 days**. 
+1. In the toolbar, select **Edit**, then **Create new**.
+1. Select **VisBuilder** in the **New Visualizations** window.
+1. In the **Data Source** dropdown list, select `opensearch_dashboards_sample_data_flights`.
+1. Drag the fields **Cancelled** and **FlightDelay** to the y-axis column.
+1. Drag the field **FlightDelayType** to the x-axis column.
+1. Select **Save** and name the visualization in the **Title** field.
+1. Select **Save and return**. The bar chart is added as the last panel to the dashboard.
+
+<img src="{{site.url}}{{site.baseurl}}/images/viz-panel-quickstart.png" alt="Creating a visualization panel" width="500"/>
+
+# Interacting with data
+
+Interactive dashboards allow you analyze data more deeply and filter it several ways. In Dashboards, you can interact directly with data on a dashboard by using dashboard-level filters. For example, continuing with the preceding dashboard, you can filter to show delays and cancellations for a specific airline.
+
+## Try it: Interacting with the sample flight data
+
+1. On the **[Flights] Airline Carrier** panel, select **OpenSearch-Air**. The dashboard updates automatically.
+1. Select **Save** to save the customized dashboard.
+
+Alternatively, you can apply filters using the dashboard toolbar:
+
+1. In the dashboard toolbar, select **Add filter**.
+1. From the **Field**, **Operator**, and **Value** dropdown lists, select **Carrier**, **is**, and **OpenSearch-Air**, respectively.
+
+    <img src="{{site.url}}{{site.baseurl}}/images/edit-filter.png" alt="Edit field interface" width="400"/>
+
+1. Select **Save**. The dashboard updates automatically.
+
+The result is the following dashboard: 
+
+  <img src="{{site.url}}{{site.baseurl}}/images/interact-filter-dashboard.png" alt="Dashboard view after applying Carrier filter" width="800"/>
+
+# Taking the next steps
+
+**Visualize data**. Learn more about data visualizations in OpenSearch Dashboards. Go to [**Visualize**]({{site.url}}{{site.baseurl}}/dashboards/visualize/viz-index/).
+
+**Create dashboards.** Learn more about visualizing data in OpenSearch Dashboards. Go to [**Dashboard**](add link-content in development).
+
+**Explore data**. Learn more about exploring data in OpenSearch Dashboards. Go to [**Discover**]({{site.url}}{{site.baseurl}}/dashboards/discover/). 
```

Comment:
```suggestion
- To learn more about exploring data in OpenSearch Dashboards, see [**Discover**]({{site.url}}{{site.baseurl}}/dashboards/discover/). 
```

Reply:
Changed to a simplified bulleted list. Not sure if you need to remove the empty lines.

Reply:
Change made. Removed empty lines